test/cpp/end2end/end2end_test.cc

```diff
@@ -218,8 +226,39 @@ class TestServiceImpl : public ::grpc::testing::EchoTestService::Service {
   explicit TestServiceImpl(const grpc::string& host)
```

Comment:
I did the merge. There were three failures in "Basic tests" but they are unrelated. The interop tests aren't run (the interop tests seem to be failing for a while now: https://grpc-testing.appspot.com/job/gRPC_interop_pull_requests/)